### ISSUE TITLE
Room list sorting should consider only message event

### ISSUE THREAD

**Issue by bmarty** [vector-im/riotX-android#480]
https://matrix.to/#/!tTIucwUZLRtKnXeurb:matrix.org/$156545601147886BnHbK:sorunome.de?via=matrix.org&via=privacytools.io&via=feneas.org

> did anyone mention yet that the sorting is a tad odd: it sorts by last event, ratehr than last message event. Meaning if someone changes their avatar all rooms they are in are bumped up to the top

**Comment by ajs124** [comment]
There should also be an option to specify sorting behavior. Personally, I would like my room list to be sorted alphabetically. There has been a whole bunch of discussion on roomlist ordering over on the [Riot Web issue tracker](https://github.com/vector-im/riot-web/issues/4343), which I think applies to any IM client, including RiotX.

**Comment by bmarty** [comment]
I agree that room sorting is a sensible subject. And also agree that random ordering as it is now (because it depends on last message date of each room) is bad to find a room in a predictive way...
We have to iterate internally to improve this
